projects/storefrontlib/src/lib/cms-lib/breadcrumb/breadcrumb.module.ts

```diff
@@ -13,10 +12,10 @@ import { CmsConfig } from '@spartacus/core';
       cmsComponents: {
         BreadcrumbComponent: { selector: 'cx-breadcrumb' }
       }
-    })
+    }),
+    CmsPageTitleModule
   ],
   declarations: [BreadcrumbComponent],
-  entryComponents: [BreadcrumbComponent],
-  exports: [BreadcrumbComponent]
```

Comment:
Why we don't export `BreadcrumbComponent` anymore?

Reply:
Since breadcrumb is configured as a CMS component, there's no need to export it. We would only export it in case we would reuse it directly in our code base. If customers however would like to directly import it, they can still do that since we're exporting the component in our public 
 API.